scripts/compose.py

```diff
@@ -1758,6 +1772,18 @@ def start_handler(self):
         services = {}
         for service in selections:
             services.update(service.render())
+
+        # expose a list of enabled opbeans services to all opbeans services. This allows them to talk amongst each other
+        # and have a jolly good distributed time
+        enabled_opbeans_services = [k for k in services.keys()
+                                    if k.startswith("opbeans-") and k not in ("opbeans-rum", "opbeans-load-generator")]
```

Comment:
@beniwohli does this include the ports? I'm testing with opbeans java and I'm not getting the ports.